public/pages/ConfigureModel/components/FeatureAccordion/FeatureAccordion.tsx

```diff
@@ -113,6 +113,7 @@ export const FeatureAccordion = (props: FeatureAccordionProps) => {
     if (props.displayMode === 'flyout') {
       return (
         <EuiButtonIcon
+          aria-label='trash'
```

Comment:
what's the motivation of adding this? I believe currently it's showing trash icon

Reply:
I found this warning `console.warn
    EuiButtonIcon requires aria-label or aria-labelledby to be specified because icon-only
          buttons are screen-reader-inaccessible without them.` when running unit tests, so do you think this is a valid change? If not I'll remove it.